Makefile

```diff
@@ -3,36 +3,25 @@ NAME = enigma
 VERSION = 0.1.0
 BUILD = `git rev-parse HEAD`
 
-GOTOOLS = \
-	github.com/golang/dep
-
 LDFLAGS = -ldflags "-X=main.Version=$(VERSION) -X=main.Build=$(BUILD)"
 
-tools: ## Install the tools needed for development and dependency management
-	go get -u -v $(GOTOOLS)
-	# 'dep' need extra steps:
-	cd $(GOPATH)/src/github.com/golang/dep && git describe --abbrev=0 --tags | xargs git checkout && go install ./cmd/dep && git checkout master
-
 clean: ## Remove the generated binary
 	-@rm -f $(NAME)
 
-deps: ## Ensure package dependencies are available
-	@dep ensure
+download: ## download dependencies
+	@ go get -t ./...
```

Comment:
Here you can also use `go mod download`. But make sure env variable `GO111MODULES=on`